### DISCUSSION TITLE
Elektrischer Verbrauch stündlich

### DISCUSSION THREAD

**Discussion by supajo** [john30/ebusd#550] [Q&A]
### Description

Hallo, eine Frage: über die Vaillant-App in Kombination mit dem VRF900 kann man ja den elektrischen Verbrauch stündlich abfragen - sowohl für Warmasser als auch für Heizung. Sowie die Energiegewinnung über die WP.

Im automatisch geladenen Konfigfile ""vaillant/15.700.csv" können diese Werte jedoch nicht stündlich abgefragt werden.
Hier gibt's Tag, Monat, Jahr. Die Werte ändern sich hier auch nur 1x pro Tag.

Gibt's hier eine Möglichkeit, die stündlichen Werte ebenso abzugreifen? 

Konfigfiles:
08.hmu.csv
15.700.csv
e0.omu.csv

Vielen Dank :-)

**Comment by dabbelju007** [comment]
Hallo zusammen,

auch wenn das Thema schon über 4 Jahre alt ist möchte ich es noch mal nach vorne holen. Genau das würde mich grade sehr interessieren. Hat jemand einen Weg gefunden die Stromverbrauchsdaten (egal ob nach Hc und Hwc getrennt oder direkt aufsummiert) über den ebus abzufragen?

Danke und viele Grüße
Dabbelju
